psi4/driver/qcdb/basislistother.py

```diff
@@ -332,6 +337,15 @@ def load_basfam_other():
     basisfamily_list.append(basis_pcsseg3)
     basisfamily_list.append(basis_pcsseg4)
 
+    # Minix
+    basis_def2msvp = BasisFamily('def2-mSVP', zeta=2)
```

Comment:
not quite following. RHS isn't MINIX, and `basis_def2msvp` on LHS has already been used above.

Reply:
Oops.